invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -165,6 +166,18 @@ export class RDMDepositForm extends Component {
               </AccordionField>
 
               <AccordionField
+                includesPaths={{
+                  "metadata": [
+                    "resource_type",
+                    "title",
+                    "additional_titles",
+                    "publication_date",
+                    "creators",
+                    "description",
+                    "additional_descriptions",
+                    "rights"
+                  ]
```

Comment:
how about:
```suggestion
                includesPaths={[
                    "metadata.resource_type",
                    "metadata.title",
                    "metadata.additional_titles",
                    "metadata.publication_date",
                    "metadata.creators",
                    "metadata.description",
                    "metadata.additional_descriptions",
                    "metadata.rights"
                  ]
```
I know it is repetitive, but the structure would be consistent with the other fields in formik, we pass full  paths as a string and decouple them by `.` separator